styles/tokens/dropdown.json

```diff
@@ -139,7 +139,7 @@
         },
         "color-active": {
           "comment": "The color of the Dropdown trigger on hover, active, focus.",
-          "value": "{black.value}",
+          "value": "{purple.value}",
```

Comment:
verify that we added the color link changes also affect the dropdown trigger

Reply:
checked. it's in there